File: data/java/FIND_MAXIMUM_PRODUCT_OF_A_TRIPLET_IN_ARRAY_TRANSLATED.py
====================
def find_maximum_product_of_a_trap_let_in_array ( arr , n ) :
    if n < 3 :
        return - 1
    max_product = int ( - 1 )
    for i in range ( n - 2 ) :
        for j in range ( i + 1 , n - 1 ) :
            for k in range ( j + 1 , n ) :
                max_product = max ( max_product , arr [ i ] * arr [ j ] * arr [ k ] )
    return max_product

File: data/python/FIND_MAXIMUM_PRODUCT_OF_A_TRIPLET_IN_ARRAY_TRANSLATED.java
====================
static int f_gold ( int [ ] arr , int n ) {
  if ( n < 3 ) return - 1 ;
  int maxProduct = - ( Integer . MAX_VALUE - 1 ) ;
  for ( int i = 0 ;
  i <= n - 2 ;
  i ++ ) {
    for ( int j = i + 1 ;
    j <= n - 1 ;
    j ++ ) {
      for ( int k = j + 1 ;
      k <= n ;
      k ++ ) {
        maxProduct = Math . max ( maxProduct , arr [ i ] * arr [ j ] * arr [ k ] ) ;
      }
    }
  }
  return maxProduct ;
}

